randovania/data/json_data/prime3.txt

```diff
@@ -491,7 +503,7 @@ Asset id: 12525204153859996508
       Trivial
 
 > Dock to Data Storage; Heals? False
-  * Normal Door to Data Storage/Dock to Conduit B
+  * Ice Missile Blast Shield to Data Storage/Dock to Conduit B
   > Dock to Substation West
       Trivial
   > Dock to Cargo Dock B
```

Comment:
For Generator A (below): Missing connections.

Reply:
Generator A doesn't connect to anywhere else, so probably irrelevant